client/src/app/components/identity/tree/IdentityTreeItem.tsx

```diff
@@ -0,0 +1,301 @@
+                                xmlns="http://www.w3.org/2000/svg"
+                            >
+                                <path
+                                    d="M1 0C1 32 21 32 21 64M21 64C21 84.8889 21 54.9259 21 64Z"
+                                    stroke="#EEEEEE"
+                                    strokeWidth="2"
+                                />
+                            </svg>
+                        </div>
+                    )}
+
+                    {this.props.nested && (
+                        <Fragment>
+                            <div className="lower-left-straight-line" />
+                            <div className="upper-left-straight-line" />
+                            {!this.props.firstMsg && <div className="upper-right-straight-line" />}
+                            {!this.props.lastMsg && <div className="lower-right-straight-line" />}
+                            {this.props.firstMsg && (
+                                <div className="upper-curved-line">
+                                    <svg
+                                        width="22"
+                                        height="72"
+                                        viewBox="0 0 22 72"
+                                        fill="none"
+                                        xmlns="http://www.w3.org/2000/svg"
+                                    >
+                                        <path
+                                            d="M1 0C1 32 21 32 21 64M21 64C21 84.8889 21 54.9259 21 64Z"
+                                            stroke="#EEEEEE"
+                                            strokeWidth="2"
+                                        />
+                                    </svg>
+                                </div>
+                            )}
+                        </Fragment>
+                    )}
+                    {!this.props.nested && (
+                        <div className="msg-icon-left">
+                            <IdentityMsgStatusIcon status="integration" />
+                        </div>
+                    )}
+                    {this.props.nested && (
+                        <div className="msg-icon-right">
+                            <IdentityMsgStatusIcon status="diff" />
+                        </div>
+                    )}
+                    <div
+                        className={classNames("content row", {
+                            "push-right": this.props.nested
+                        })}
+                    >
+
+                        {/* --------- Title and timestamp --------- */}
+                        <div>
+                            <p className="title">{this.shortenMsgId(this.props.messageId ?? "")}</p>
+                            {this.props.content?.created ? (
+                                <p className="time-stamp">
+                                    {moment(this.props.content?.created).format("MMM D  h:m:s a")}
```

Comment:
Would these be better as the `updated` field, otherwise they will always show the same value ?